Assignment 5.Rmd

```diff
@@ -66,17 +66,21 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
 #Upload new data
 
-D2 <- 
+D2 <- read.csv("intelligent_tutor_new.csv", header = TRUE)
+#Generate predicted advice for new students based on tree generated from old students
 
 #Generate predicted advice using the predict() command for new students based on tree generated from old students
 
-D2$prediction <- 
+D2$prediction <- predict(score_ctree, D2)
 
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
-
+```{r}
+#All students in the new sample achieved a score of 1, and therefore would need no action. Therefore the accuracy of your model on the new data would be:
+mean(ifelse(D2$prediction == "no action", 1, 0))
```

Comment:
Nice work! You wrote good comments and interpretations. It is also good to report whether you think the model is acceptable based on the accuracy and how you would like to improve this in your future work.